compiler/il/OMRBlock.hpp

```diff
@@ -478,6 +483,8 @@ class OMR_EXTENSIBLE Block : public TR::CFGNode
     * Flag functions end
     */
 
+   AbsEnvStatic *_absEnv;
```

Comment:
we don't want to have this pointer in every block of every compile - that is going to be too expensive - we need to find another way to get hold of it.

Reply:
Sounds good! We can talk about this in a couple of minutes.